harness/types/capabilities.test.ts

```diff
@@ -0,0 +1,34 @@
+import { Sdks } from './sdks'
+
+describe('SDKCapabilities Unit Tests', () => {
+    afterEach(() => {
+        jest.resetModules()
+    })
+
+    it('should load capabilities from env variables', () => {
+        process.env.SDK_CAPABILITIES = 'edgeDB,cloud,clientCustomData,sse'
+        process.env.SDKS_TO_TEST = 'NodeJS,Python'
+        const { SDKCapabilities } = jest.requireActual('../capabilities')
+        expect(SDKCapabilities).toEqual({
+            NodeJS: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+            Python: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+        })
+    })
+
+    it('should load capabilities from env variables as JSON', () => {
+        process.env.SDK_CAPABILITIES =
+            '["edgeDB","cloud","clientCustomData","sse"]'
```

Comment:
why bother supporting both formats?

Reply:
basically copied this / wanted to match the `getSDKs()` method: 
<img width="706" alt="image" src="https://github.com/DevCycleHQ/test-harness/assets/1219069/17854b1b-2981-4685-9293-69e5404a69e3">